pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/TokenUtils.java

```diff
@@ -55,35 +55,43 @@ public static <T extends GenericToken<T>> T nthFollower(T token, int n) {
      */
     // test only
     public static <T extends GenericToken<T>> T nthPrevious(T startHint, T anchor, int n) {
-        if (startHint.compareTo(anchor) >= 0) {
-            throw new IllegalStateException("Wrong left hint, possibly not left enough");
-        }
-        if (n <= 0) {
-            throw new IllegalArgumentException("Offset can't be less than 1");
-        }
+        validateInput(startHint, anchor, n);
```

Comment:
this line needs to be under try-catch block to catch known exceptions: IllegalStateException, IllegalArgumentException